PRIVACY_POLICY.ja.md

```
@@ -73,10 +73,13 @@ Hackers' Pub は、ソフトウェアエンジニアおよびテクノロジー
 GitHub、Mastodon、個人ウェブサイトなど、プロフィールに登録した外部サービスへの
 リンクと、その暗号学的な認証状態。
 
-### プッシュ通知（iOS）
+### プッシュ通知
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)